src/app/NavBar.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+import logo from "src/app/assets/logos/Cobuild_Logo.svg";
+
+const Navbar: React.FC = () => {
```

Comment:
React.FC is an outdated construct; we should probably use the modern style of writing components. It looks like you don't have any props here, so it would just be a `const Navbar = () => {}`